packages/frontend/src/contexts/AppContext/useTxHistory.ts

```diff
@@ -1,8 +1,10 @@
-import { useEffect, useCallback, Dispatch, SetStateAction } from 'react'
+import { useEffect, useCallback, Dispatch, SetStateAction, useRef, useState } from 'react'
+import { useQuery } from 'react-query'
 import { useLocalStorage } from 'react-use'
 import Transaction from 'src/models/Transaction'
 import find from 'lodash/find'
 import { filterByHash, sortByRecentTimestamp } from 'src/utils'
+import _ from 'lodash'
```

Comment:
suggestion: import lodash functions individually to create a smaller build so it doesn't import all other functions

```suggestion
import isFunction from 'lodash/isFunction'
import cloneDeepWith from 'lodash/cloneDeepWith'
```